planbot.tf

```diff
@@ -92,23 +89,26 @@ resource "google_cloudfunctions_function" "run_plan_bot" {
   available_memory_mb = 256
 
   event_trigger {
-    event_type = "providers/cloud.pubsub/eventtypes/topic.publish"
+    event_type = "google.pubsub.topic.publish"
     resource = google_pubsub_topic.run_plan_bot.name
   }
+
   source_archive_bucket = google_storage_bucket.plan_bot_function_storage.name
   source_archive_object = google_storage_bucket_object.plan_bot_zip.name
   timeout = 60
-  entry_point = "run_plan_bot"
+  entry_point = "run_plan_bot_event_handler"
   max_instances = 1
   labels = {
     // here so that the deploy trick above keeps working
     deployment-tool = "cli-gcloud"
   }
 
   environment_variables = {
+    REPLIED_TO_PATH = "gs://${google_storage_bucket.plan_bot_other_storage.name}/${google_storage_bucket_object.plan_bot_plans_replied_to.name}"
   }
```

Comment:
When we trust this deployed version and want to turn on posting, all we should need to do is add `SEND_REPLIES=true` here :D

Reply:
Great work!